File: hw/unda/U2/HW13Crud/Taxes.py/tax.py
# tax.py

class Tax:
    def computeTax(self, base_price: float, percentage: float, charged: bool) -> float:
        if charged:
            return base_price * (percentage / 100)
        else:
            return 0.0

    def testComputeIce(self, base_price: float) -> float:
        return 0.0

File: ws/caizapanta/u3/WS28UnitTest/Taxes/src/main/java/ec/edu/espe/taxes/controller/Tax.java
package ec.edu.espe.taxes.controller;

/**
 *
 * @author Caizapanta Tammy SoftCrafters
 */
public class Tax {

    public float computeTax(float basePrice, float percentage, boolean charged) {
        float ivaValue = 0.0f;
        if (!charged) {
            ivaValue = 0.0f;
        } else {
            ivaValue = basePrice * percentage / 100;
        }
        return ivaValue;
    }
    public float testComputeIce(float basePrice){
        System.out.println("computeIce");
        return basePrice;
    }
}
